resources/apigateway-restapis.go

```diff
@@ -74,6 +78,7 @@ func (f *APIGatewayRestAPI) Properties() types.Properties {
 	properties.
 		Set("APIID", f.restAPIID).
 		Set("Name", f.name).
-		Set("Version", f.version)
+		Set("Version", f.version).
+		Set("CreatedDate", f.createdDate)
```

Comment:
```suggestion
		Set("CreatedTime", r.createdTime.Format(time.RFC3339))
```

Also for the other resource, thanks